modules/graphics/mrblib/renderer.rb

```diff
@@ -0,0 +1,9 @@
+module Moon
+  class Renderer
+    def render(shader, vbo, texture, transform, mode)
```

Comment:
It did, and then mruby spat an error out about requiring 5 parameters, I don't think it likes constants as default parameters
```ruby
def render(shader, vbo, texture, transform, mode = GL2::GL_TRIANGLE_STRIP)
# ...
end
```